ci/prepare_env_buildkite.sh

```diff
@@ -3,9 +3,24 @@
 set -e
 
 export APP_VERSION_CODE=$((BUILDKITE_BUILD_NUMBER))
-# get newest tags from origin
-git fetch
-# strip the first char as that should always be "v" (as tags should be in the format "vX.X.X")
-description="$(git describe --tags --always)"
-export APP_VERSION_NAME=${description:1}
+
+# Buildkite branch equals to tag name if build was triggered by tag
+if [[ $BUILDKITE_BRANCH  =~ ^v[0-9]+.* ]]; then
+    export APP_VERSION_NAME=${BUILDKITE_BRANCH:1}
+else
+  export RC_INDICATOR=""
+  version="$(./gradlew -q pV | tail -1)"
```

Comment:
Calling the task via `printVersion` instead of `pV` would be more clear. And also I would've found it when I was looking for where printVersion is used.